### PR TITLE
Simplify the docker run documentation and run quickly with docker-com…

### PR DESCRIPTION
**Pull request by Tinywan**
…pose

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,52 +1,81 @@
...
+$ docker pull bitnami/etcd:3.3.13-r80
+```
+
+#### Step 2: pull this apisix image
+
+```
+$ docker pull iresty/apisix:latest
+```
+> if you are in the mainland China, use mirror
+>  `docker pull registry.cn-beijing.aliyuncs.com/tinywan/apisix:alpine`
```
Comment (moonming) [inline README.md]: I don’t think I should pull a personal image.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+version: "3"
+
+services:
+  apisix:
+    # image: iresty/apisix:latest
+    # if you are in the mainland China, please use China mirror
+    image: registry.cn-beijing.aliyuncs.com/tinywan/apisix:alpine
```
Comment (moonming) [inline docker-compose.yml]: ditto

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+version: "3"
```
Comment (moonming) [inline docker-compose.yml]: what's the different with https://github.com/apache/incubator-apisix-docker/blob/master/example/docker-compose.yml?

File: README.md [hunk 4]
```diff
@@ -1,52 +1,81 @@
...
+#### Step 1: pull this etcd image
+
+```
+$ docker pull bitnami/etcd:3.3.13-r80
+```
+
+#### Step 2: pull this apisix image
+
+```
+$ docker pull iresty/apisix:latest
```
[the remark this answers is not in the document]
Comment (Tinywan) [reply README.md]: Please tell me the official image

File: README.md (hunk 1 above)
Comment (Tinywan) [reply README.md]: Due to the wall, many official images can not be pulled, deleted

File: docker-compose.yml (hunk 2 above)
Comment (Tinywan) [reply docker-compose.yml]: Due to the wall, many official images can not be pulled, deleted

**Pull request closed without merging**